src/qt/forms/optionsdialog.ui

```diff
@@ -319,6 +319,16 @@
          </property>
         </widget>
        </item>
+       <item>
+        <widget class="QCheckBox" name="enableServer">
+         <property name="toolTip">
+          <string>Accept command line and JSON-RPC commands.</string>
+         </property>
+         <property name="text">
+          <string>Enable RPC server</string>
```

Comment:
Let's add a mnemonic shortcut as well as a translator comment:
```suggestion
          <string extracomment="An Options window setting to enable the RPC server.">Enable &amp;RPC server</string>
```

Reply:
I agree with @hebasto [comment](https://github.com/bitcoin-core/gui/pull/416#issuecomment-912438215). This is probably a better fit under the main tab as it is not related to the P2P network. As we collect more settings, this could be a good candidate to move into a new "Advanced" or "Developer" tab.